app/test_main.py

```diff
@@ -1 +1,54 @@
-# write your code here
+from app.main import can_access_google_page
+from unittest.mock import patch
+
+
+@patch("app.main.valid_google_url")
+@patch("app.main.has_internet_connection")
+def test_can_access_google_page_accessible(
+        mock_internet: patch,
+        mock_valid_url: patch
+) -> None:
+    mock_internet.return_value = True
+    mock_valid_url.return_value = True
+
+    result = can_access_google_page("https://www.google.com")
+    assert result == "Accessible"
```

Comment:
No need to create new variable

```suggestion
    assert can_access_google_page("https://www.google.com") == "Accessible"
```

Reply:
also add message with error description, for example:

```suggestion
    assert can_access_google_page("www.google.com/") == "Accessible", (
        "Should be accessible when URL is valid and there is an internet connection."
    )
```

Reply:
the same everywhere